kodilla-hibernate/src/main/java/com/kodilla/hibernate/manytomany/Company.java

```diff
@@ -5,6 +5,15 @@
 import java.util.ArrayList;
 import java.util.List;
 
+
+@NamedNativeQuery(
+        name = "Company.retrieveCompanyWithTreeChars",
+        query = "SELECT * FROM COMPANIES" +
+                " WHERE (COMPANIES LIKE : %keyword%)  = :KEYWORD",
```

Comment:
Składnia nie jest poprawna. `:KEYWORD` powinnaś użyć po `LIKE`

Reply:
Nie rozumiem co tu mam zrobic, takie coś jest w treści zadania "frazy field LIKE :ARG, gdzie za field należy wstawić nazwę pola z tabeli, natomiast do :ARG należy wstawić fragment nazwiska otoczony z obu stron znakami "%" (co zastępuje dowolny ciąg znaków)"
Poproszę bardziej szczegółowa podpowiedz.

Reply:
Chodzi o samą składnię po `WHERE`: masz `WHERE (COMPANIES LIKE : %keyword%)  = :KEYWORD`, powinno być po prostu `WHERE COMPANIES LIKE : %:KEYWORD%`.
Rozkładając to na czynniki pierwsze: (COMPANIES LIKE : %keyword%) - sprawdzasz, czy pole `COMPANIES` ma wartość zawierającą słowo `keyword` (ale nie argument :KEYWORD, tylko po prostu napis keyword). Całe to wyrażenie jest typu bool (true albo false), dalej porównujesz to do swojego słowa kluczowego. To nigdy nie zadziała ;)